types/ssz.go

```diff
@@ -81,6 +83,34 @@ func (b SSZTransactions) HashTreeRoot() ([32]byte, error) {
 	return ssz.HashWithDefaultHasher(b)
 }
 
+// SSZWithdrawals --
+type SSZWithdrawals []*capella.Withdrawal
+
+func (b SSZWithdrawals) GetTree() (*ssz.Node, error) {
```

Comment:
is there a reason why we don't use a pointer here?

Reply:
lets derive a task for this